stdlib/system/system.flux

```diff
@@ -1,4 +1,13 @@
 package system
 
 
+// time is a function that returns the current system time
+//
+// ## Example
+// ```
+// import "system"
+//
+// data
+//   |> set(key: "processed_at", value: string(v: system.time() ))
```

Comment:
this example is not valid as the `data` var is undefined, perhaps a better example would be to use `array.from`

```
import "system"
import "array"

array.from(rows:[{time: system.time()}])
```